crates/autopilot/src/domain/settlement/trade/math.rs

```diff
@@ -134,9 +134,15 @@ impl Trade {
 
     /// Total fee (protocol fee + network fee). Equal to a surplus difference
     /// before and after applying the fees.
-    pub fn fee_in_sell_token(&self) -> Result<eth::SellTokenAmount, Error> {
+    ///
+    /// Denominated in SELL token
```

Comment:
Do we need this clarification here since the function name is self-explanatory already?